src/ontology/mondo-edit.obo

```diff
@@ -584341,6 +584343,17 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1030003
+name: mycoplasma infection
```

Comment:
Mycoplasma should have a capital letter because it refers to the genus name

Reply:
done